pkg/provider/resource_nrs_alert_condition.go

```diff
@@ -79,6 +84,23 @@ func NRSAlertConditionCreate(resourceData *schema.ResourceData, meta interface{}
 	return nil
 }
 
+
+func NRSAlertConditionImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+    s := strings.Split(d.Id(), ":")
```

Comment:
Can we document the fact that we're overloading the ID field for imports in the README?